blog-preview-card/styles.css

```diff
@@ -0,0 +1,107 @@
+body {
+  background-color: hsl(47, 88%, 63%);
+  font-family: "Figtree";
```

Comment:
You are not using this font anywhere - you will need to add a `@font-face` that defines the font and an url to it (local url)